spaces/domains.json

```
@@ -124,5 +124,6 @@
   "vote.perlinx.finance": "perlinx.eth",
   "vote.defigeek.xyz": "defigeek.eth",
   "ambassador.synthetix.io": "snxambassador.eth",
+  "vote.bamboodefi.com": "bamboodefi.eth"
```

You miss a comma at the end of the line.
```suggestion
  "vote.bamboodefi.com": "bamboodefi.eth",
```